regions_www/tabs/about_body.html

```diff
@@ -14,7 +14,7 @@
 
       <div class="col-md-12 text-left" style="padding-top:10px;">
         <h3><a name="about"></a>About the Propensity to Cycle Tool</h3>
-        <p> This is the online home of the open source transport planning system, the Propensity to Cycle Tool (PCT).The PCT is released under the <a href="/www/licence.html" target="_blank">Affero GPL</a>: it is free to use, copy and modify, e.g. to create versions for new cities and states. Please cite the journal article Lovelace et al. (<a href="https://www.jtlu.org/index.php/jtlu/article/view/862" target="_blank">2017</a>) and/or Goodman et al. (<a href="https://www.sciencedirect.com/science/article/pii/S2214140518301257" target="_blank">2019</a>) if you use the PCT in your work.</p>
+        <p> This is the online home of the open source transport planning system, the Propensity to Cycle Tool (PCT). The PCT is released under the <a href="/www/licence.html" target="_blank">Affero GPL</a>: it is free to use, copy and modify, e.g. to create versions for new cities and states. Please cite the journal article Lovelace et al. (<a href="https://www.jtlu.org/index.php/jtlu/article/view/862" target="_blank">2017</a>) and/or Goodman et al. (<a href="https://www.sciencedirect.com/science/article/pii/S2214140518301257" target="_blank">2019</a>) if you use the PCT in your work.</p>
```

Comment:
I cannot see any change here.

Reply:
@Robinlovelace 
As the commit says, I've added a space. Find it just before `The PCT is released under the  <a href="/www/licence.html" target="_blank">Affero GPL`

Reply:
Ah yes. Cheers.